src/Codata/Delay/Properties.agda

```diff
@@ -48,3 +49,64 @@ module _ {a b c} {A : Set a} {B : Set b} {C : Set c} where
  map-unfold-fusion f n s with n s
  ... | Sum.inj₁ s′ = later λ where .force → map-unfold-fusion f n s′
  ... | Sum.inj₂ b  = now Eq.refl
+
+
+⇓-unique : ∀ {a} → {A : Set a} →
+           {d : Delay A ∞} →
+           (d⇓₁ : d ⇓) → (d⇓₂ : d ⇓) →
+           d⇓₁ ≡ d⇓₂
+⇓-unique {d = now s} (now s) (now s) = Eq.refl
+⇓-unique {d = later d'} (later l) (later r) =
+  Eq.cong later (⇓-unique {d = force d'} l r)
+
+
+bind⇓-injₗ : ∀ {a} {A B : Set a} →
+             {d : Delay A ∞} {f : A → Delay B ∞} →
+             bind d f ⇓ → d ⇓
+bind⇓-injₗ {d = now s} foo = now s
+bind⇓-injₗ {d = later s} (later foo) =
+  later (bind⇓-injₗ foo)
+
+
```

Comment:
We only put single empty lines between similar proofs. It might be nice though if you added a section heading for these proofs concerning `⇓` though, e.g. see the ones in `Data.List.Properties`

https://github.com/agda/agda-stdlib/blob/b52e3e5d6185f119211cfdf8c63d1a0f9b2502a8/src/Data/List/Properties.agda#L75-L78